services/merak-topo/handler/topo_deploy.go

```diff
@@ -254,38 +257,38 @@ func Topo_deploy(k8client *kubernetes.Clientset, topo database.TopologyData) err
 					Tolerations:                   tol,
 				},
 			}
-		} else if strings.Contains(node.Name, "cgw") {
-			l["Type"] = "configgw"
-
-			log.Printf("assign cgw to node %v", k8snodes[0])
-
-			newPod = &corev1.Pod{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:   node.Name,
-					Labels: l,
-				},
-				Spec: corev1.PodSpec{
-					InitContainers: init_containers,
-					Containers: []corev1.Container{
-						{
-							Name:            "cgw",
-							Image:           GW_IMAGE,
-							ImagePullPolicy: "IfNotPresent",
-							Command:         []string{"sleep", "100000"},
-							SecurityContext: &sc,
-						},
-					},
-					RestartPolicy:                 "OnFailure",
-					NodeName:                      k8snodes[0],
-					TerminationGracePeriodSeconds: &grace_period,
-					Tolerations:                   tol,
-				},
-			}
-			if len(k8snodes) > 1 {
-				k8snodes = k8snodes[1:]
-				log.Printf("unassigned nodes %v", k8snodes)
-
-			}
+			// } else if strings.Contains(node.Name, "cgw") {
+			// 	l["Type"] = "configgw"
+
+			// 	log.Printf("assign cgw to node %v", k8snodes[0])
+
+			// 	newPod = &corev1.Pod{
+			// 		ObjectMeta: metav1.ObjectMeta{
```

Comment:
@zzengcs If this section of code need to be preserved, please give some comment before the code, otherwise please remove them.

Reply:
Resolved by adding comment notes